### PR TITLE
fix readstateview layout issue

### PR DESCRIPTION
**Pull request by adrian09h**
# Submit a pull request

## ReadStateView Layout issue
When add(delete) reaction, the layout has an issue.
- Before:

- After:

### CODE REVIEW

File: library/src/main/java/com/getstream/sdk/chat/adapter/AttachmentViewHolderMedia.java [hunk 1]
```diff
@@ -150,10 +150,20 @@ private void configMediaAttach() {
 //        iv_media_thumb.setOnLongClickListener(this);
         if (!TextUtils.isEmpty(attachUrl) && !attachUrl.contains("https:"))
             attachUrl = "https:" + attachUrl;
-        Glide.with(getContext())
-                .load(attachUrl)
-                .placeholder(R.drawable.stream_placeholder)
-                .into(iv_media_thumb);
+
+        if (attachment.getType().equals(ModelType.attach_giphy)){
```
Comment (tbarbugli) [inline library/src/main/java/com/getstream/sdk/chat/adapter/AttachmentViewHolderMedia.java]: looks unrelated

Comment (adrian09h) [reply library/src/main/java/com/getstream/sdk/chat/adapter/AttachmentViewHolderMedia.java]: yeah. Glide loads giphy without indicating asGif().
